CharlieBackend.Business/Services/CourseService.cs

```diff
@@ -62,23 +64,22 @@ public async Task<Result<CourseDto>> UpdateCourseAsync(long id, UpdateCourseDto
             {
                 if (updateCourseDto == null)
                 {
-                    return Result<CourseDto>.GetError(ErrorCode.ValidationError, "CourseDto is null");
+                    return Result<CourseDto>.GetError(ErrorCode.ValidationError, "invalid course model");
```

Comment:
Missing model and wrong model i think i different reasons. Can you pls rename to "Missing"